examples/simulated/plot_riemannian_gaussian.py

```diff
@@ -0,0 +1,77 @@
+"""
+=====================================================================
+Sample from the Riemannian Gaussian distribution in the SPD manifold
+=====================================================================
+
+Spectral embedding of samples from the Riemannian Gaussian distribution
+with different centerings and dispersions.
+
+"""
+# Authors: Pedro Rodrigues <pedro.rodrigues@melix.org>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pyriemann.embedding import Embedding
+from pyriemann.datasets import sampling
```

Comment:
put sample_gaussian_spd and generate_random_spd_matrix in pyriemann/datasets/__init__.py  so you can do

```suggestion
from pyriemann.datasets import generate_random_spd_matrix, sample_gaussian_spd
```